releases/2.5/release-notes.asciidoc

```diff
@@ -15,6 +15,89 @@ Release numbers follow http://semver.org[Semantic Versioning].
 
 toc::[]
 
+[[release-2.5.0-alpha2]]
+== *Release 2.5.0.Alpha2* _(November 10th 2023)_
+
+See the https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12317320&version=12415492[complete list of issues].
+
+=== Kafka compatibility
+
+This release has been built against Kafka Connect 3.6.0 and has been tested with version 3.6.0 of the Kafka brokers.
+See the https://kafka.apache.org/documentation/#upgrade[Kafka documentation] for compatibility with other versions of Kafka brokers.
+
+
+=== Upgrading
+
+Before upgrading any connector, be sure to check the backward-incompatible changes that have been made since the release you were using.
+
+When you decide to upgrade one of these connectors to 2.5.0.Alpha2 from any earlier versions,
+first check the migration notes for the version you're using.
+Gracefully stop the running connector, remove the old plugin files, install the 2.5.0.Alpha2 plugin files, and restart the connector using the same configuration.
+Upon restart, the 2.5.0.Alpha2 connectors will continue where the previous connector left off.
+As one might expect, all change events previously written to Kafka by the old connector will not be modified.
+
+If you are using our container images, then please do not forget to pull them fresh from https://quay.io/organization/debezium[Quay.io].
+
+
+=== Breaking changes
+
+MySQL 5.7 is end of life.
+Debezium is no longer officially supporting it or testing with it.
+Further support is on best-effort basis only (https://issues.redhat.com/browse/DBZ-6874[DBZ-6874]).
+
+Configuration option `metadata.location` was renamed to `metadata.source` for CloudEvents converter (https://issues.redhat.com/browse/DBZ-7060[DBZ-7060]).
+
+MongoDB default connection mode is switched from `replica_set` to `sharded`.
+This is the first step in removing the `replica_set` mode completely.
+Please note that the offsets will be invalidated and snapshot triggered for existing deployments unless the configuration option is explicitly set (https://issues.redhat.com/browse/DBZ-7108[DBZ-7108]).
```

Comment:
@Naros This needs to be super-highlighted in release notes. @jcechace Would rprovide more details if needed.